CHANGES.rst

```diff
@@ -4,6 +4,8 @@ Changes
 4.2.0 (unreleased)
 ------------------
 
+- Add support for Python 3.5
```

Comment:
All other changelog entries that I can see (after expanding the context once because I'm lazy) end with a period.